src/coreclr/hosts/unixcoreconsole/CMakeLists.txt

```diff
@@ -31,8 +31,4 @@ if(NOT CLR_CMAKE_PLATFORM_ANDROID)
   )
 endif()
 
-add_dependencies(coreconsole
-    coreclr
```

Comment:
>why do we build corerun and coreconsole during cross components build? 
I do not think this is true. cross-components build only builds dll/exe listed in file https://github.com/dotnet/coreclr/blob/master/crosscomponents.cmake